source/wp-content/themes/wporg-developer-blog/parts/footer.html

```diff
@@ -0,0 +1,63 @@
+
+            <!-- wp:columns {"style":{"spacing":{"blockGap":{"top":"var:preset|spacing|30","left":"var:preset|spacing|40"}}}} -->
+            <div class="wp-block-columns">
+                <!-- wp:column -->
+                <div class="wp-block-column">
+                    <!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|10"}},"layout":{"type":"default"}} -->
+                    <div class="wp-block-group">
+                        <!-- wp:heading {"level":3,"style":{"elements":{"link":{"color":{"text":"var:preset|color|blueberry-2"}}},"typography":{"fontStyle":"normal","fontWeight":"700"}},"textColor":"blueberry-2","fontSize":"normal","fontFamily":"inter"} -->
+                        <h3 class="wp-block-heading has-blueberry-2-color has-text-color has-link-color has-inter-font-family has-normal-font-size" style="font-style:normal;font-weight:700"><strong><a href="https://developer.wordpress.org/news/how-to-contribute/">Learn how to contribute</a></strong></h3>
+                        <!-- /wp:heading -->
+
+                        <!-- wp:paragraph {"fontSize":"normal"} -->
+                        <p class="has-normal-font-size">Share your knowledge with fellow WordPress developers.</p>
+                        <!-- /wp:paragraph -->
+                    </div>
+                    <!-- /wp:group -->
+                </div>
+                <!-- /wp:column -->
+
+                <!-- wp:column -->
+                <div class="wp-block-column">
+                    <!-- wp:group {"style":{"spacing":{"blockGap":"var:preset|spacing|10"}},"layout":{"type":"default"}} -->
+                    <div class="wp-block-group">
+                        <!-- wp:heading {"level":3,"style":{"elements":{"link":{"color":{"text":"var:preset|color|blueberry-2"}}}},"textColor":"blueberry-2","fontSize":"normal","fontFamily":"inter"} -->
+                        <h3 class="wp-block-heading has-blueberry-2-color has-text-color has-link-color has-inter-font-family has-normal-font-size"><strong><a href="https://developer.wordpress.org/news/tips-and-guidelines-for-writers/">Review tips and guidelines</a></strong></h3>
+                        <!-- /wp:heading -->
+
+                        <!-- wp:paragraph -->
+                        <p>Everything you need to know about writing for the Blog.</p>
+                        <!-- /wp:paragraph -->
+                    </div>
+                    <!-- /wp:group -->
+                </div>
+                <!-- /wp:column -->
+            </div>
+            <!-- /wp:columns -->
+        </div>
+        <!-- /wp:column -->
+
+        <!-- wp:column {"style":{"spacing":{"blockGap":"var:preset|spacing|20"}}} -->
+        <div class="wp-block-column">
+            <!-- wp:heading {"style":{"typography":{"fontStyle":"normal","fontWeight":"400"}},"fontSize":"large"} -->
+            <h2 class="wp-block-heading has-large-font-size" style="font-style:normal;font-weight:400">Subscribe to the Blog</h2>
```

Comment:
Secondary to providing great content and surfacing it, this is likely the second most important goal for the homepage. I would consider pulling it out into a dedicated component. 

Alternatively, this section should at least communicate what I can expect content-wise if I sign up answering the question "Why should I sign up"?

Reply:
Yeah, there needs to be greater emphasis on getting folks to subscribe to the blog. I have been discussing this with the Editorial team, and some ideas are in the works for early next year. I think we should launch with this design and iterate based on the upcoming promotional efforts.